src/main/scala/mesosphere/marathon/state/Container.scala

```diff
@@ -37,16 +36,16 @@ object Container {
 
   case class Mesos(volumes: Seq[Volume] = Seq.empty) extends Container
 
-  case class Docker(
+  case class DockerDocker(
```

Comment:
Imo we can go with `Docker` vs `MesosDocker`. I'm getting gosebumps when I read _DockerDocker_